os/tools/mkbinheader.py

```diff
@@ -84,18 +84,38 @@
     else : # Not supported.
         bin_type = 0
 
+    # Calculate RAM size
+    # Dynamic RAM size : user input of argv[6]
+    # Static RAM size : Extract from size command in linux(ONLY for elf type)
+    if bin_type == BIN :
+        os.system('size ' + tinyara_path + ' > static_ram_file')
+    elif bin_type == ELF :
+        os.system('size ' + file_path + ' > static_ram_file')
```

Comment:
You don't need to name "_file" and it looks strange name, "static_ram_file".
`static_ram_usage` or `static_ram_estimation` or something else